common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -254,3 +254,17 @@ def test_prefered_language_field(self):
             u'',
             [u'Pushto', u''],
         )
+
+    def test_connected_accounts(self):
```

Comment:
These should exist only if the feature flag for 3rd party auth is enabled, correct?